src/goto-checker/all_properties_verifier.h

```diff
@@ -35,10 +35,6 @@ class all_properties_verifiert : public goto_verifiert
 
   resultt operator()() override
   {
-    // have we got anything to check? otherwise we return PASS
-    if(!has_properties_to_check(properties))
-      return resultt::PASS;
-
```

Comment:
The commit message suggests that this code must be put somewhere else, but then the commit only removes code?

Reply:
incremental_goto_checker is already doing that.